test/notificationSpec.js

```diff
@@ -56,7 +87,7 @@ describe('Notifications', function () {
       delete this.notificationNoConditionFulfillment.id
       yield this.request()
         .post('/notifications')
-        .send(this.notificationNoConditionFulfillment)
+        .send(signNotification(this.notificationNoConditionFulfillment))
```

Comment:
I don't think you should sign every notification in the tests. It seems like it adds an unnecessary step to every single test. What about instead just having one set of tests where it requires the notifications to be signed and then tests just the functionality you want?